### ISSUE TITLE
NoSuchMethodError Crash on 1.21

### ISSUE THREAD

**Issue by Amcio** [jpenilla/better-fabric-console#43]
On 1.21 with version 1.1.10 the server crashes when sending any command to the console.

```
Encountered an unexpected exception
java.lang.NoSuchMethodError: 'net.kyori.adventure.text.Component net.minecraft.class_2561.asComponent()'
        at net.minecraft.server.dedicated.DedicatedServer.sendSystemMessage(DedicatedServer.java:753) ~[?:?]
        at net.minecraft.commands.CommandSourceStack.sendSuccess(CommandSourceStack.java:306) ~[?:?]
        at net.minecraft.server.commands.ListPlayersCommand.format(ListPlayersCommand.java:40) ~[?:?]
        at net.minecraft.server.commands.ListPlayersCommand.listPlayers(ListPlayersCommand.java:29) ~[?:?]
        at net.minecraft.server.commands.ListPlayersCommand.lambda$register$0(ListPlayersCommand.java:20) ~[?:?]
        at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.2.9.jar:?]
        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:29) ~[?:?]
        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:13) ~[?:?]
        at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8) ~[?:?]
        at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:8) ~[?:?]
        at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:107) ~[?:?]
        at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:379) ~[?:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:309) ~[?:?]
        at net.minecraft.commands.Commands.performPrefixedCommand(Commands.java:299) ~[?:?]
        at net.minecraft.server.dedicated.DedicatedServer.handleConsoleInputs(DedicatedServer.java:318) ~[?:?]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:300) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:912) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:697) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:281) ~[server-intermediary.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
```

**Comment by jpenilla** [comment]
I can't reproduce this with just better-fabric-console and fabric API. I would guess some other mod is causing adventure-platform-fabric's Mixins to fail to apply, but it's hard to say with the limited information you've shared.

**Comment by Amcio** [comment]
This is the crash log with information about the mods installed, I removed some already but at this point i'm not sure why would any of these interfere with loading mixins. Could you take a look?

```
Time: 2024-06-26 19:53:17
Description: Exception in server tick loop

java.lang.NoSuchMethodError: 'net.kyori.adventure.text.Component net.minecraft.class_2561.asComponent()'
	at net.minecraft.server.dedicated.DedicatedServer.sendSystemMessage(DedicatedServer.java:753)
	at net.minecraft.commands.CommandSourceStack.sendSuccess(CommandSourceStack.java:306)
	at net.minecraft.server.commands.ListPlayersCommand.format(ListPlayersCommand.java:40)
	at net.minecraft.server.commands.ListPlayersCommand.listPlayers(ListPlayersCommand.java:29)
	at net.minecraft.server.commands.ListPlayersCommand.lambda$register$0(ListPlayersCommand.java:20)
	at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73)
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:29)
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:13)
	at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8)
	at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:8)
	at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:107)
	at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:379)
	at net.minecraft.commands.Commands.performCommand(Commands.java:309)
	at net.minecraft.commands.Commands.performPrefixedCommand(Commands.java:299)
	at net.minecraft.server.dedicated.DedicatedServer.handleConsoleInputs(DedicatedServer.java:318)
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:300)
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:912)
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:697)
	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:281)
	at java.base@21.0.3/java.lang.Thread.run(Thread.java:1583)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.21
	Minecraft Version ID: 1.21
	Operating System: Linux (amd64) version 5.15.0-112-generic
	Java Version: 21.0.3, Ubuntu
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Ubuntu
	JVM Flags: 1 total; -Xmx4G
	Fabric Mods: 
		appleskin: AppleSkin 3.0.2+mc1.21
		audioplayer: AudioPlayer 1.21-1.10.2
			admiral: Admiral 0.4.4+1.20.5
		better-fabric-console: Better Fabric Console 1.1.10
			adventure-platform-fabric: adventure-platform-fabric 5.14.0
				net_kyori_adventure-api: adventure-api 4.17.0
				net_kyori_adventure-key: adventure-key 4.17.0
				net_kyori_adventure-platform-api: adventure-platform-api 4.3.3
				net_kyori_adventure-text-logger-slf4j: adventure-text-logger-slf4j 4.17.0
				net_kyori_adventure-text-minimessage: adventure-text-minimessage 4.17.0
				net_kyori_adventure-text-serializer-ansi: adventure-text-serializer-ansi 4.17.0
				net_kyori_adventure-text-serializer-gson: adventure-text-serializer-gson 4.17.0
				net_kyori_adventure-text-serializer-json: adventure-text-serializer-json 4.17.0
				net_kyori_adventure-text-serializer-plain: adventure-text-serializer-plain 4.17.0
				net_kyori_ansi: ansi 1.0.3
				net_kyori_examination-api: examination-api 1.3.0
				net_kyori_examination-string: examination-string 1.3.0
				net_kyori_option: option 1.0.0
			com_typesafe_config: config 1.4.1
			io_leangen_geantyref_geantyref: geantyref 1.3.11
			net_fabricmc_mapping-io: mapping-io 0.6.1
			org_fusesource_jansi_jansi: jansi 2.4.1
			org_jline_jline: jline 3.26.2
			org_jline_jline-terminal-jansi: jline-terminal-jansi 3.26.2
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		betterstats: Better Statistics Screen 3.11.2+fabric-1.21
		bluemap: BlueMap 5.2
		charm: Charm 7.0.28
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
			com_moandjiezana_toml_toml4j: toml4j 0.7.2
		cloth-config: Cloth Config v15 15.0.127
			cloth-basic-math: cloth-basic-math 0.6.1
		creativecore: CreativeCore 2.11.34
			net_neoforged_bus: bus 7.2.0
		entityculling: EntityCulling 1.6.6
		fabric-api: Fabric API 0.100.3+1.21
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.67+b5597344d1
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.28+6fc22b99d1
			fabric-block-api-v1: Fabric Block API (v1) 1.0.22+0af3f5a7d1
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.10+6573ed8cd1
			fabric-command-api-v1: Fabric Command API (v1) 1.2.48+f71b366fd1
			fabric-commands-v0: Fabric Commands (v0) 0.2.65+df3654b3d1
			fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.13+b5597344d1
			fabric-convention-tags-v1: Fabric Convention Tags 2.0.14+7f945d5bd1
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.3.1+8a3b5617d1
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.29+0af3f5a7d1
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.23+6a6dfa19d1
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 20.2.8+16c4ae25d1
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.0+6fc22b99d1
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.12+6fc22b99d1
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.10+e633f883d1
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.52+6573ed8cd1
			fabric-item-api-v1: Fabric Item API (v1) 11.0.0+afdfc921d1
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.1+cb5ced13d1
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.11+8f3583aed1
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.10+6573ed8cd1
			fabric-message-api-v1: Fabric Message API (v1) 6.0.13+6573ed8cd1
			fabric-networking-api-v1: Fabric Networking API (v1) 4.2.0+ab7edbacd1
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.1.11+d1321076d1
			fabric-particles-v1: Fabric Particles (v1) 4.0.2+6573ed8cd1
			fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.9+6573ed8cd1
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.0.22+ab7edbacd1
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.48+73761d2ed1
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.6+b5597344d1
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.2.1+d153f344d1
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.1.4+cb5ced13d1
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.79+b5597344d1
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.1.14+b5597344d1
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.0.12+6573ed8cd1
		fabric-language-kotlin: Fabric Language Kotlin 1.11.0+kotlin.2.0.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 2.0.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.24.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.8.1
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.8.1
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.3
		fabricloader: Fabric Loader 0.15.11
			mixinextras: MixinExtras 0.3.5
		ferritecore: FerriteCore 7.0.0
		jade: Jade 15.0.4+fabric
		java: OpenJDK 64-Bit Server VM 21
		journeymap: Journeymap 1.21-6.0.0-beta.8
			ar_com_hjg_pngj: pngj 2.1.0
			commonnetworking: Common Network 1.0.11-1.21
			journeymap-api-fabric: JourneyMap API 2.0.0-1.21-SNAPSHOT
		minecraft: Minecraft 1.21
		nochatreports: No Chat Reports 1.21-v2.8.0
		notenoughanimations: NotEnoughAnimations 1.7.4
		placeholder-api: Placeholder API 2.4.0-pre.2+1.21
		skinlayers3d: 3d-Skin-Layers 1.6.6
		sound_physics_remastered: Sound Physics Remastered 1.21-1.4.3
		strange: Strange 7.0.13
		tcdcommons: TCD Commons API 3.11.1+fabric-1.21
		voicechat: Simple Voice Chat 1.21-2.5.17
			fabric-api-base: Fabric API Base 0.4.42+6573ed8c90
			fabric-command-api-v2: Fabric Command API (v2) 2.2.27+6a6dfa1990
		yet_another_config_lib_v3: YetAnotherConfigLib 3.5.0+1.21-fabric
			com_twelvemonkeys_common_common-image: common-image 3.10.0
			com_twelvemonkeys_common_common-io: common-io 3.10.0
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
	Server Running: true
	Active Data Packs: vanilla, fabric, audioplayer, charm, fabric-convention-tags-v2, strange
	Available Data Packs: audioplayer, bundle, charm, fabric, fabric-convention-tags-v2, strange, trade_rebalance, vanilla
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	World Seed: -4895606492541891167
	Is Modded: Definitely; Server brand changed to 'fabric'
```

**Comment by jpenilla** [comment]
Fixed in 1.2.0
